cmd/gpu_nfdhook/labeler_test.go

```diff
@@ -264,12 +277,7 @@ func TestLabeling(t *testing.T) {
 			if tc.expectedRetval == nil && !reflect.DeepEqual(labeler.labels, tc.expectedLabels) {
 				t.Errorf("test %v label mismatch with expectation:\n%v\n%v\n", tc.name, labeler.labels, tc.expectedLabels)
 			}
-			for filename := range tc.capabilityFile {
-				os.Remove(path.Join(root, filename))
-			}
-			for filename := range tc.sysfsfiles {
-				os.Remove(path.Join(sysfs, filename))
-			}
+			tc.deleteFiles(t, sysfs, root)
```

Comment:
I think it would be simpler to create test files in a sub-root specific to a testcase. Then `defer os.RemoveAll(root)` could take care of all the temp files.

Reply:
Done. My apologies for the multiple pushes.